src/main/java/io/vlingo/actors/World.java

```diff
@@ -323,4 +325,12 @@ private void startRootFor(final Stage stage, final Logger logger) {
         null,
         logger);
   }
+
+  public void registerDynamic(final String name, final Object dep) {
```

Comment:
please move these into public area above around the other register??? methods in alphabetical order... sorry, but there is a method to my madness.